ansible/roles_ocp_workloads/ocp4_workload_minio/templates/route.yaml.j2

```diff
@@ -0,0 +1,17 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: minio
+  namespace: "{{ ocp4_workload_minio_project }}"
+spec:
+#  host: minio-gpte-minio.apps.cluster-3afa.sandbox105.opentlc.com
```

Comment:
Should remove commented out code like this when not being used. It confuses at a glance and can accidentally be uncommented breaking things.

Reply:
Done